### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#24] [Ideas]
All aspects of design.

**Comment by seekanddefine** [comment]
Ideas for icon use:

https://www2.deloitte.com/au/en/pages/about-deloitte/articles/our-blog.html?utm_source=blog&utm_medium=wp&utm_campaign=exc-blog-2019&utm_content=blogmigration#.WFYrDhuLReU

![image](https://user-images.githubusercontent.com/38668794/123855824-ef5ea600-d8e5-11eb-9289-2f8efa4fd702.png)

I like how these images appear almost as hybrid icons/emojis/icons. Some of the don't make semantic sense in my mind, but they are pretty and the three dimensionality really catches my eye on the background.

**Comment by seekanddefine** [comment]
An icon set that @avidrucker offered with the idea of it becoming a functional/structural item based on skills acquired on the learning path. 

![image](https://user-images.githubusercontent.com/38668794/123856206-6c8a1b00-d8e6-11eb-9b80-f0bf29d7baf6.png)

I love the simplicity and pops of color, although I'm not sure Iike how some of them are filled in with color and most have a white background.

**Comment by avidrucker** [comment]
What are some good visual metaphors for learning bits, nodes, streams, etc.?
